src/App.tsx

```diff
@@ -15,35 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_ALPHABET = 'alphabet';
+const SORT_FIELD_LENGTH = 'length';
+
+interface FilterParams<T> {
+  sort: keyof T | '';
+  reversed: boolean;
+}
+
+function getPrepearedGoods<T>(
```

Comment:
There is a typo in the function name `getPrepearedGoods`. It should be `getPreparedGoods` to correct the spelling.